.vscodeignore

```diff
@@ -1,24 +1,24 @@
-# This excludes files during npm pack

```

Comment:
You should test the ACR Build stuff, it's got HTML that might be affected by path changes (or ask CTI to test it): https://microsoft.sharepoint.com/teams/azcode/_layouts/OneNote.aspx?id=%2Fteams%2Fazcode%2FSiteAssets%2Fazcode%20Notebook&wd=target%28Testing.one%7CD12CBB51-9002-4371-B958-B88191DF5CF5%2FRun%20a%20Build%20Task%7C0458D37A-AB49-418F-9A95-21D671C61B2C%2F%29
onenote:https://microsoft.sharepoint.com/teams/azcode/SiteAssets/azcode%20Notebook/Testing.one#Run%20a%20Build%20Task&section-id={D12CBB51-9002-4371-B958-B88191DF5CF5}&page-id={0458D37A-AB49-418F-9A95-21D671C61B2C}&end

Reply:
Done. See screenshots above

Reply:
Oh, I see you already did... :-)